python/lsst/afw/image/_exposureSummaryStats.py

```diff
@@ -285,6 +299,27 @@ def update_schema(cls, schema: Schema) -> None:
             type="F",
             doc="Maximum distance of an unmasked pixel to its nearest model psf star (pixel).",
         )
+        schema.addField(
+            "effTime",
+            type="F",
+            doc="Effective exposure time calculated from psfSigma, skyBg, and "
+            "zeroPoint (seconds)."
```

Comment:
It's possible/preferred to add a unit to the schema as well.  (This was an oversight with the original version).

Reply:
Ok, I'll add units for all quantities that have them (also keeping units in the docstring).